src/sources/journald.rs

```diff
@@ -298,7 +303,7 @@ impl JournaldSource {
             // valid and non-filtered event.
             while batch.events.is_empty() {
                 let item = tokio::select! {
-                    _ = &mut shutdown => return false,
+                    _ = &mut *shutdown => return false,
```

Comment:
Why even take another mutable reference at all? Should just be able to do `_ = shutdown => return false,`?